### DISCUSSION TITLE
How can i make {class: ...} work with the `prettier-plugin-tailwindcss`?

### DISCUSSION THREAD

**Discussion by pabloruizetc** [vanjs-org/van#182] [Q&A] [answered]
I would like to sort tailwindcss classes using the `prettier-plugin-tailwindcss` ([ref](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/main/README.md)).

**Comment by Tao-VanJS** [comment, marked as answer]
For your specific request, I think you can [sort classes in function calls](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/main/README.md#sorting-classes-in-function-calls).
